calico_node/startup/startup.go

```diff
@@ -602,21 +579,18 @@ func ensureGlobalBGPConfig(c *client.Client, key, def string) error {
 // not required in the format string.
 func message(format string, args ...interface{}) {
 	fmt.Printf(format+"\n", args...)
-	log.Infof(format, args...)
 }
 
 // warning() prints a warning to screen and to log.  A newline terminator is
 // not required in the format string.
 func warning(format string, args ...interface{}) {
 	fmt.Printf("WARNING: "+format+"\n", args...)
```

Comment:
whouldn't this start with a `\n` if it's Printf? I'm thinking if previous print line didn't end with a `\n` this would end up in the same line